nifi-docs/src/main/asciidoc/user-guide.adoc

```diff
@@ -2922,6 +2922,34 @@ key5=c6FzfnKm7UR7xqI2NFpZ+fEKBfSU7+1NvRw+XWQ9U39MONWqk5gvoyOCdFR1kUgeg46jrN5dGXk
 
 Each line defines a key ID and then the Base64-encoded cipher text of a 16 byte IV and wrapped AES-128, AES-192, or AES-256 key depending on the JCE policies available. The individual keys are wrapped by AES/GCM encryption using the **root key** defined by `nifi.bootstrap.sensitive.key` in _conf/bootstrap.conf_.
 
+===== KeyStoreKeyProvider
+The `KeyStoreKeyProvider` implementation reads from an encrypted keystore using the configured password to load AES Secret Key entries.
+
+The provider supports the following Keystore Types:
+
+* BCFKS
+* PKCS12
```

Comment:
We are discouraging use of JKS?

Reply:
JKS does not support storage of Secret Key entries, so attempting to run `keytool -genseckey -storetype JKS` throws a KeyStoreException.  PKCS12 is also the default KeyStore Type starting in Java 9.